app/views/defence_requests/_table_body.html.erb

```diff
@@ -0,0 +1,20 @@
+<% @defence_requests.each_with_index do |def_req, ind| %>
+  <tr id=<%= "dr_#{def_req.id}" %>>
```

Comment:
This will render as `<tr id=dr_100>` - need to change it to this:

``` erb
<tr id="dr_<%= def_req.id %>">
```

alternatively you could use [`content_tag_for`][1]:

``` erb
<% content_tag_for(:tr, def_req) do %>
  <td><%= def_req.solicitor_type %></td>
  …
<% end %>
```

Only problem would be that id would be `defence_request_100` not `dr_100`.

[1]: http://api.rubyonrails.org/classes/ActionView/Helpers/RecordTagHelper.html#method-i-content_tag_for